packages/apps/ability-uniswap-swap/src/lib/schemas.ts

```diff
@@ -1,82 +1,53 @@
-import { SchemaValidationError } from '@lit-protocol/vincent-ability-sdk';
 import { z } from 'zod';
-import { bundledVincentPolicy as spendingLimitBundledPolicy } from '@lit-protocol/vincent-policy-spending-limit';
 
 export const abilityParamsSchema = z.object({
-  ethRpcUrl: z
-    .string()
-    .describe(
-      'An Ethereum Mainnet RPC Endpoint. This is used to check USD <> ETH prices via Chainlink.',
-    ),
   rpcUrlForUniswap: z
     .string()
     .describe(
-      'An RPC endpoint for any chain that is supported by the @uniswap/sdk-core package. Must work for the chain specified in chainIdForUniswap.',
-    ),
-  chainIdForUniswap: z
-    .number()
-    .describe('The chain ID to execute the transaction on. For example: 8453 for Base.'),
-
-  tokenInAddress: z
-    .string()
-    .describe(
-      'ERC20 Token address to sell. For example 0x4200000000000000000000000000000000000006 for WETH on Base.',
+      'An RPC endpoint for any chain that is supported by the @uniswap/sdk-core package. Must work for the chain specified in the signed Uniswap quote.',
```

Comment:
In my testing, the Lit Action always times out when using the Base RPC provide by the Lit nodes (it's a public endpoint), so we must provide the RPC URL for now

Reply:
Adam asked the node ops today to get a base RPC url in asap, so hopefully we can remove this soon.